experimental/traffic-portal/package.json

```diff
@@ -80,26 +82,30 @@
     "@angular-eslint/template-parser": "13.1.0",
     "@angular/cli": "^13.2.0",
     "@angular/compiler-cli": "^13.2.0",
+		"@nightwatch/schematics": "^0.1.9",
     "@types/argparse": "^2.0.2",
     "@types/chart.js": "^2.9.34",
     "@types/express": "^4.17.0",
     "@types/jasmine": "~3.6.0",
     "@types/jasminewd2": "~2.0.3",
+    "@types/nightwatch": "2.0.1",
     "@types/node": "^14.17.34",
     "@typescript-eslint/eslint-plugin": "^5.10.0",
     "@typescript-eslint/parser": "^5.10.0",
+    "chromedriver": "^100.0.0",
     "codelyzer": "^6.0.0",
     "eslint": "^8.2.0",
     "eslint-plugin-import": "^2.25.3",
     "eslint-plugin-jsdoc": "^37.0.3",
     "eslint-plugin-prefer-arrow": "^1.2.3",
+    "geckodriver": "3.0.1",
     "jasmine-core": "^3.10.1",
     "karma": "~6.3.2",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coverage": "~2.2.0",
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "^1.5.0",
-    "protractor": "~7.0.0",
+    "nightwatch": "2.0.0-beta.3",
```

Comment:
should we really be using a beta version?

Reply:
`2.1.0` breaks the tests and the [nightwatch schematic explicitly](https://github.com/nightwatchjs/nightwatch-schematics/blob/d69342bae71afb4f25918b004b88a540d685b845/src/schematics/nightwatch/utility/util.ts#L158) adds the [`next` version](https://www.npmjs.com/package/nightwatch?activeTab=versions) (2.0.0-beta.3) explicitly. Also `2.1.0` currently needs `chromedriver:99` _which is wrong_ (it's not even at a LTS release of chrome), leading me to believe these tags are not correct.

Reply:
Wonderful. I figured it was something like that, I was just hoping I was wrong.